components/win-users/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "habitat_win_users"
+version = "0.0.0"
+authors = ["Matt Wrock <matt@mattwrock.com>"]
+description = "Habitat library for win32 account api calls"
+build = "../build.rs"
```

Comment:
yeah I just copied this from another crate that did not have its own build script. I'm tempted to bring over the elevation check but was thinking of converting it to ffi.